### ISSUE TITLE
Icon wrong size in toolbar on Retina macOS

### ISSUE THREAD

**Issue by gingerbeardman** [el1t/uBlock-Safari#135]
### Describe the issue

Icon wrong size in toolbar on Retina macOS

### Screenshot in which the issue can be seen

<img width="107" alt="screen shot 2018-08-22 at 12 07 36" src="https://user-images.githubusercontent.com/49612/44460174-047a3d00-a604-11e8-9a71-cd701612d3f5.png">

### Steps for anyone to reproduce the issue

1. Use a Retina Mac
2. Update to latest
3. Look at toolbar icon

### Your settings

- MacOS version: 10.13.6 (17G65)
- Safari version:	11.1.2 (13605.3.8)
- uBlock Origin version: 1.16.0

**Comment by volcbs** [comment]
Try to turn your ubo on. Your ubo extension is off.

**Comment by gingerbeardman** [comment]
Oh, wow, is this a recent change?

This is really not clear.

Thanks

**Comment by volcbs** [comment]
Not really. Twas already there since ubo was ported to Safari. It's safari-only icon though.

**Comment by gingerbeardman** [comment]
I think an empty shield would make more sense than no shield.
